src/python/pants/engine/legacy/graph.py

```diff
@@ -717,6 +727,7 @@ def _eager_fileset_with_spec(
       glob_match_error_behavior=glob_match_error_behavior,
       # We validate that _every_ filesystem spec is valid.
       conjunction=GlobExpansionConjunction.all_match,
+      description_of_origin="file specs",
```

Comment:
Very open to rewords for this in particular. Per https://github.com/pantsbuild/pants/pull/9010#discussion_r370861981, we likely should not be using the phrase `filesystem specs` to end users. But, on the other hand, we do have the option `--specs-file`. 

I think `file specs` strikes a balance between internal implementation detail vs. language we want users to know (`specs`)? Any suggestions?

Currently, this reads: 
```
▶ ./pants cloc2 src/fake.txt
18:57:49 [WARN] Unmatched glob from file specs: "src/fake.txt"
```

Reply:
I like "cli arguments" more than "file specs", I think. 

My sole concern is the `--specs-file` option. It's a little confusing to read "cli arguments" when they were provided by that option. At this point of execution, we've discarded the information of whether they used this option or not, so can't generate a more useful message. 

I'm going to wait to change to see if anyone else has recommendations, and if not, I'll change to "cli arguments".

Reply:
"file arguments"? Meaning that these are cli arguments which happen to be file specs. I don't really like the word "spec" too much, but that might be just me.

Reply:
We have a winner! Thanks Borja.